### PR TITLE
Graduate Active Balance Cache Feature

### PR DESCRIPTION
**Pull request by nisdas**
**What type of PR is this?**

Feature Graduation

**What does this PR do? Why is it needed?**

This makes our active balance cache the permanent default in prysm and removes the ability to disable it.

**Which issues(s) does this PR fix?**

N.A

**Other notes for review**

### CODE REVIEW

**Review by rkapka — DISMISSED** [review]

**Review by james-prysm — APPROVED** [review]
LGTM, radek's approval went stale after merging develop.

**Pull request merged**
